.github/workflows/ci.yml

```
@@ -138,3 +148,21 @@ jobs:
         env:
           # Should match 'name:' up above
           JOB_NAME: 'macOS (${{ matrix.python }})'
+
+  # https://github.com/marketplace/actions/alls-green#why
+  check:  # This job does nothing and is only used for the branch protection
+
+    if: always()
+
+    needs:
+      - Windows
+      - Ubuntu
+      - macOS
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Decide whether the needed jobs succeeded or failed
+        uses: re-actors/alls-green@release/v1
```

People generally use mutable tags so there's no security around this, in case you care.  Could pin to the commit hash instead after a review if anyone is concerned about it.

---

Here's a convenient `I care` button if anybody needs this:
```suggestion
        uses: re-actors/alls-green@13b4244b312e8a314951e03958a2f91519a6a3c9
```